conjure-java-client-verifier/src/test/java/com/palantir/verification/AutoDeserializeTest.java

```diff
@@ -189,8 +189,8 @@ public void runTestCaseJersey() throws Error, NoSuchMethodException {
     private Optional<Error> expectFailureJersey(Method method) {
         try {
             Object result = method.invoke(testServiceJersey, index);
-            return Optional.of(new AssertionError(
-                    String.format("Result should have caused an exception but deserialized to: %s", result)));
+            return Optional.of(new AssertionError(String.format(
+                    "Result should have caused an exception but deserialized to: %s", result)));
```

Comment:
I preferred the original formatting here, but it doesn't make much difference either way